mmil.raw.html

```diff
@@ -8032,7 +8027,7 @@
 
 <TR>
   <TD>sumrblem</TD>
-  <TD>~ isumrblem</TD>
+  <TD>~ sumrbdclem</TD>
```

Comment:
Why is this theorem mentioned here, but not isummolem2, for example?

Reply:
This hasn't been especially systematic. There are places where lemmas match quite closely between set.mm and iset.mm and I guess sometimes they have been listed. In general, though, we don't list them.

If the lemma seems like it could be used outside the proof of a single main theorem, that would be a reason to list it, but that doesn't seem to be true of sumrblem so I've pushed up a change to remove it from this list.